stacks/config.js

```diff
@@ -100,3 +101,44 @@ export function getGitInfo () {
     branch: git.branch('.')
   }
 }
+
+/**
+ * @param {import('@serverless-stack/resources').Stack} stack
+ */
+export function setupSentry (stack) {
```

Comment:
maybe we should check stage here instead? thoughts?

Reply:
yes, we're using the config file set config depending on the stage already, so let's do the `if (!app.local)` check once here rather than in every stack

Reply:
done ✨